doc/kube-setup-data-ingestion-job.md

```diff
@@ -0,0 +1,16 @@
+The data-ingestion-job is specific to DataSTAGE.
+
+To prep: 
+Fill out the config file with creds here: `$(gen3_secrets_folder)/g3auto/data-ingestion-job/data_ingestion_job_config.json`
+Place a newline-separated list of phs IDs here: `$(gen3_secrets_folder)/g3auto/data-ingestion-job/phsids.txt`
+Optionally place a "data_requiring_manual_review.tsv" file here: `$(gen3_secrets_folder)/g3auto/data-ingestion-job/data_requiring_manual_review.tsv`
+Optionally place a "genome_file_manifest" here: `$(gen3_secrets_folder)/g3auto/data-ingestion-job/genome_file_manifest.csv`
+
+Usage:
+`gen3 kube-setup-data-ingestion-job CREATE_GOOGLE_GROUPS <bool> CREATE_GENOME_MANIFEST <bool>`
+
+If CREATE_GENOME_MANIFEST is true, the genome file manifest is required to live in `g3auto/data-ingestion-job/`.
```

Comment:
This is a little confusing, I would assume that "CREATE_GENOME_MANIFEST" being true means that the script will create it for me. Maybe change to "USE_PROVIDED_GENOME_MANIFEST"?

Reply:
or is this supposed to say `if CREATE_GENOME_MANIFEST is false`?

Reply:
good catch! it shouldve said false haha